package.json

```diff
@@ -1,22 +1,33 @@
 {
-  "name": "iale",
-  "private": true,
-  "version": "0.0.0",
-  "type": "module",
-  "scripts": {
-    "dev": "vite",
-    "build": "tsc && vite build",
-    "preview": "vite preview"
-  },
-  "dependencies": {
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0"
-  },
-  "devDependencies": {
-    "@types/react": "^18.0.22",
-    "@types/react-dom": "^18.0.7",
-    "@vitejs/plugin-react": "^2.2.0",
-    "typescript": "^4.6.4",
-    "vite": "^3.2.0"
-  }
-}
\ No newline at end of file
+    "name": "iale",
+    "private": true,
+    "version": "0.0.0",
+    "type": "module",
+    "scripts": {
+        "dev": "vite",
+        "build": "tsc && vite build",
+        "preview": "vite preview"
+    },
+    "dependencies": {
+        "bootstrap": "^5.2.2",
+        "eslint-config-airbnb-base": "^15.0.0",
```

Comment:
I don't think you need it + it should be in devDeps

Reply:
why not and what of these?

Reply:
Read about https://www.npmjs.com/package/eslint-config-airbnb and https://www.npmjs.com/package/eslint-config-airbnb-base and tell me the difference

Reply:
1)exporting 3 configurations for usage / only 2
2)With react plugins / without react plugins